vavr/src-gen/main/java/io/vavr/CheckedFunction1.java

```diff
@@ -298,6 +298,20 @@ default <V> CheckedFunction1<V, R> compose(CheckedFunction1<? super V, ? extends
         Objects.requireNonNull(before, "before is null");
         return v -> apply(before.apply(v));
     }
+
+    /**
+     * Returns a composed function that first applies the {@linkplain Function} {@code before1} to the
+     * 1st argument and then applies this CheckedFunction1 to the result.
+     *
+     * @param <S> argument type of before1
+     * @param before1 the function applied before this
+     * @return a function composed of before1 and this
+     * @throws NullPointerException if before1 is null
+     */
+    default <S> CheckedFunction1<S, R> compose1(Function1<? super S, ? extends T1> before1) {
```

Comment:
This is now (almost) a duplicate of `compose`, but including it does make the API more consistent.